modules/exploits/multi/http/liferay_java_unmarshalling.rb

```diff
@@ -70,23 +73,25 @@ def check
       )
     end
 
-=begin
-    Building the Liferay-Portal header:
-      https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/util/ReleaseInfo.java
-    Liferay-Portal header data:
-      https://github.com/liferay/liferay-portal/blob/master/release.properties
-
-    Example GET / response:
-      HTTP/1.1 200
-      [snip]
-      Liferay-Portal: Liferay Community Edition Portal 7.2.0 CE GA1 (Mueller / Build 7200 / June 4, 2019)
-      [snip]
-=end
+    #     Building the Liferay-Portal header:
```

Comment:
Since we're not using block comments, the extra indentation here is even more unnecessary.

Reply:
Agreed; I think Rubocop was playing it safe here when automatically fixing this.

Reply:
I'm on the fence about block comments, but they're really useful for pasting in structured information.

Reply:
Also agreed! The docs shed a bit of light on the decision:

- https://rubocop.readthedocs.io/en/stable/cops_style/#styleblockcomments
- https://rubystyle.guide/#no-block-comments

Reply:
This was not fixed.